<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Connect</class>
 <widget class="QDialog" name="Connect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>126</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Соединение с сервером</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelServerName">
       <property name="text">
        <string>Имя сервера</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="lineEditServerName">
       <property name="text">
        <string>WIN-S2NUV43JAS1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelLogin">
       <property name="text">
        <string>Имя пользователя</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEditLogin"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelPassword">
       <property name="text">
        <string>Пароль</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="lineEditPassword"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonConnect">
     <property name="text">
      <string>Соединение</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
